### ISSUE TITLE
Legal declaration 

### ISSUE THREAD

**Issue by gavinwye** [hmrc/design-patterns#73]
## Screenshots

Examples from various prototypes

![image](https://cloud.githubusercontent.com/assets/868772/22469670/b9b80756-e7c4-11e6-800e-4361394a33d9.png)

![image](https://cloud.githubusercontent.com/assets/868772/22469700/ccd1d01a-e7c4-11e6-9671-d5c247cb8f03.png)

![image](https://cloud.githubusercontent.com/assets/868772/22469721/e211b6f2-e7c4-11e6-81a8-f7afac75feb3.png)

![image](https://user-images.githubusercontent.com/868772/29518787-ab65c9d4-8672-11e7-9bce-f9a87826a5c4.png)

![image](https://user-images.githubusercontent.com/868772/29518801-c29aa0a2-8672-11e7-9442-22ae496862ab.png)

## Other links
GOV.UK hackpad: https://designpatterns.hackpad.com/Declarations-RB89Ux2CqUn

**Comment by KateEvans** [comment]
![paperless](https://user-images.githubusercontent.com/11850209/29521629-308f59f8-867e-11e7-9f97-6a25066dc9bf.png)
![quarterly reporting](https://user-images.githubusercontent.com/11850209/29521628-3088ca66-867e-11e7-99fb-1bf47523063c.png)

**Comment by gavinwye** [comment]
## Initial meeting to discuss the pattern

### Attendees 
- @stevenaproctor 
- @KateEvans 
- @chrismoorembe
- @gavinwye

## Declarations pattern 

### Start point
We have an un-substantiated standpoint that we should not be using checkboxes in services. We need input from people who have done user research on declarations to understand how users react to them.

### Questions to answer 
- do we really need a declaration?
- if we need a declaration when should they be used?
- should declarations appear on the same page a check your answers?
- what's the user need for a declaration?
- should we do a co-design session with legal and security people?
- if users are authenticated do we treat declarations differently to if they are authenticated?

There is a big overlap with terms and conditions.

## Terms and conditions
Content designers often strip out a lot of information contained in terms and conditions, replacing them with easier to understand information.

### Start point
Terms and conditions should be:
- short & concise
- written in plain English

### Teams currently working on this at HMRC
- SDES
- subscriptions
- API team 

## Actions
- [ ] find a representative from security and legal workshops [@KateEvans]
- [ ] set up a workshop with interested parties [@gavinwye]
- [ ] build, collate, contribute an example of what we think we should be doing [@stevenaproctor]
- [ ] arrange another meeting on 21st September [@gavinwye]
- [ ] start a branch in the prototype to document the pattern [@gavinwye]
- [ ] create a new issue for Terms and Conditions [@gavinwye]

**Comment by stevenaproctor** [comment]
@gavinwye 
@KateEvans 
@chrismoorembe

I had a look through the [Information Commissioner's site]((https://ico.org.uk/for-organisations/data-protection-reform/overview-of-the-gdpr/key-areas-to-consider) to find the information Paper was trying to link to. I found this stuff about consent that applies to what we were talking about on Monday.

"Consent under the GDPR must be a freely given, specific, informed and unambiguous indication of the individual’s wishes. There must be some form of clear affirmative action – or in other words, a positive opt-in – consent cannot be inferred from silence, pre-ticked boxes or inactivity. Consent must also be separate from other terms and conditions, and you will need to provide simple ways for people to withdraw consent. Public authorities and employers will need to take particular care to ensure that consent is freely given."

See I do not see why clicking a button is not "clear affirmative action".

When I was searching I came across 2 related terms: browserwrap and clickwrap.

Browserwrap is when a service has terms and conditions or a privacy policy in the footer. No attention is drawn to them and there is no action. They say something like 'by using or accessing the website you agree to and are bound by the language within its terms of service and its privacy policy as presented'.

HMRC's [terms and conditions](https://www.tax.service.gov.uk/help/terms-and-conditions) say "Access and use by you of our site constitutes your acceptance of these terms and conditions. They take effect from the date on which you first use this website. We may at any time revise these terms and conditions without notice, and therefore they should be regularly checked. The continued use of the site after a change has been made is your acceptance of the change." 

Clickwrap is when you have to click something like a checkbox or a button to agree. Both versions would normally link to terms and conditions or have them on the page.

I am not an expert, but I think under GDPR browserwrap is not allowed and everything has to have actual consent. This could be at the first time of use and every time the terms are updated. Chris mentioned banks often have a page that loads to make you aware of the change and you have to select a button to get into the site.

The [GDS service manual](https://www.gov.uk/service-manual/design/check-your-answers-pages) says:

"If you add a declaration to the end of a transaction, you must give people a means to go back and change the information they've provided. For this reason, the bottom of a 'Check your answers' page can be a good place to have a declaration."

This backs up what we were talking about not having them on a separate screen.

**Comment by olirose** [comment]
Here are the 'terms of participation' from the MTDfB sign up journey for both individuals and agents. These received legal sign off by the Deputy Director for MTDfB (policy and legal). 

Individual
![image](https://user-images.githubusercontent.com/31846170/31123116-76e3e2dc-a836-11e7-89fd-65ff7a5fbacf.png)

Agent
![image](https://user-images.githubusercontent.com/31846170/31123147-96741964-a836-11e7-9068-976865abdf4f.png)

**Comment by olirose** [comment]
Here are the 'terms of participation' from the MTDfB sign up journey for both individuals and agents. These received legal sign off by the Deputy Director for MTDfB (policy and legal). 

Individual
![image](https://user-images.githubusercontent.com/31846170/31123261-0e6e007e-a837-11e7-8733-f10bab9c5935.png)


Agent
![image](https://user-images.githubusercontent.com/31846170/31123147-96741964-a836-11e7-9068-976865abdf4f.png)
![image](https://user-images.githubusercontent.com/31846170/31123228-ec1ffbbc-a836-11e7-8817-58362c915183.png)

**Comment by olirose** [comment]
Here are the 'terms of participation' from the MTDfB sign up journey for both individuals and agents. These were signed off by the Deputy Director for MTDfB (policy and legal). 

Individual
![image](https://user-images.githubusercontent.com/31846170/31123261-0e6e007e-a837-11e7-8733-f10bab9c5935.png)

Agent
![image](https://user-images.githubusercontent.com/31846170/31123344-6fe49b60-a837-11e7-9f77-1d5117d97219.png)

**Comment by stevenaproctor** [comment]
@gavinwye Some of the declaration stuff is on the GOV.UK check your answers pattern. https://www.gov.uk/service-manual/design/check-your-answers-pages

**Comment by vickytnz** [comment]
The DWP component library includes this styling - is this the styling we should be following for our services?
![gov_uk_-_the_best_place_to_find_government_services_and_information](https://user-images.githubusercontent.com/1223264/33128637-03f4cb26-cf85-11e7-84b9-a6a524a9373f.jpg)

**Comment by jennifer-hodgson** [comment]
This is on the [GOV.UK Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/29).

**Comment by mikeash82** [comment]
Continue discussion on the [GOV.UK Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/29) backlog issue

**Comment by mikeash82** [comment]
Continue discussion on the [GOV.UK Design System backlog](https://github.com/alphagov/govuk-design-system-backlog/issues/29)
